src/types/global.d.ts

```diff
@@ -58,7 +58,7 @@ declare module "mozjexl/lib/parser/Parser" {
 }
 
 declare namespace browser.experiments.nimbus {
-  type EnrollWithFeatureConfigResult =
+  type EnrollInExperimentResult =
```

Comment:
If you move this type outside of the `declare namespace` block it should be available to the rest of the project and you shouldn't need to declare it in FeatureConfigPage.tsx or RecipeEnrollmetnPage.tsx